### PR TITLE
feat:Added avatar and image components

### PR DESCRIPTION
**Pull request by Hardiegogo**
Added avatars:

- large avatar
- medium avatar
- small avatar
- extra small avatar

Added images:

- responsive image
- round image
https://www.loom.com/share/38966ec113ae46f5aec6c13fc9241bad

### CODE REVIEW

File: CSS_files/components/image/image.css [hunk 1]
```diff
@@ -1,8 +1,20 @@
-img{
+.resp-img{
    width: 100%;
+   height: auto;
    object-fit: contain; 
 }
 
 .img-circle{
```
Comment (tusharkandpal) [inline CSS_files/components/image/image.css]: Can be consistent with class name by using `circle-img` or `round-img` as `resp-img`

File: docsite.html [hunk 2]
```diff
@@ -429,15 +423,138 @@ <h3>Badges</h3>
...
+                    storytellers, they can complement the content on your web-application.</p>
+
+                <p class="small-text mt-24">Responsive image</p>
+                <p class='x-small-text mt-24'>The image given below can adjust its size according to the container while
+                    maintaining its aspect ratio.</p>
+
+
+                <div class="component-wrapper p-16 br-m mt-24">
+                    <!-- Responsive image  -->
+                    <img class='resp-img'' src="http://tiny.cc/nm7ouz" alt="">
```
Comment (tusharkandpal) [inline docsite.html]: Inverted commas of class are not properly closed

**Review by tusharkandpal — CHANGES_REQUESTED** [review]

File: CSS_files/components/image/image.css (hunk 1 above)
Comment (Hardiegogo) [reply CSS_files/components/image/image.css]: I have made the necessary changes, thank you for the feedback.

File: docsite.html (hunk 2 above)
Comment (Hardiegogo) [reply docsite.html]: Corrected it, thank you.

**Review by anshikaa12 — APPROVED** [review]
looks good to merge

**Comment by Hardiegogo** [comment]
> looks good to merge

Okay, thank you.

**Pull request merged**